docs/src/pages/css-in-js/advanced/advanced.md

```diff
@@ -232,7 +232,51 @@ function App() {
 export default App;
 ```
 
-## Server Side Rendering
+## Server-side rendering
+
+Material-UI supports concurrent server-side rendering.
```

Comment:
What does concurrent mean in this context? Reacts concurrent mode isn't stable yet and I don't see any tests related to it.

Reply:
It's in the Node.js context, let's remove it, it's confusing.

Reply:
We will need to rethink the styling solution from scratch with React concurrent mode.

Reply:
> [...] reach the local minima of our current architecture [...]

I have no idea what that means.

Reply:
I believe that reaching the global minimum (best possible tradeoff) requires a lot of work. I'm interested in the quick wins we can do before v4. Imagine that the best tradeoff is to migrate to Linaria, that would be a hugh effort.
If tomorrow React starts to traverse the React tree in an unpredictibale way, our SSR solution will break. We would need to change some part of the architecture.

Reply:
The specific term is way more appropriate than the generic "global minimum". I've never heard this term with regards to software architecture.

Reply:
What's the specific term? The global vs local minimum concept can be applied to everything we do in our life: the place we work at, girlfriend we date, life habits, etc.